internal/broker/broker.go

```diff
@@ -3,15 +3,26 @@ package broker
 import (
 	"context"
 	"encoding/json"
+	"errors"
 
 	"code.cloudfoundry.org/lager"
 	xrv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
 	"github.com/pivotal-cf/brokerapi/v7/domain"
 	"github.com/pivotal-cf/brokerapi/v7/domain/apiresponses"
+	corev1 "k8s.io/api/core/v1"
 
 	"github.com/vshn/crossplane-service-broker/internal/crossplane"
 )
 
+var (
+	// ErrInstanceNotFound is an instance doesn't exist
+	ErrInstanceNotFound = errors.New("instance not found")
+	// ErrServiceUpdateNotPermitted when updating an instance
+	ErrServiceUpdateNotPermitted = errors.New("service update not permitted")
+	// ErrSLAChangeNotPermitted when updating an instance's SLA plan (only premium<->standard is permitted)
+	ErrSLAChangeNotPermitted = errors.New("SLA change not permitted")
```

Comment:
Maybe we should call this "plan change" rather than SLA change? The SLA change is actually the only one that's permitted...